content/applications/inventory_and_mrp/inventory/routes/strategies/removal.rst

```diff
@@ -1,222 +1,363 @@
-particular case, they are dispatched according to their expiration date.
+The :abbr:`FEFO (First Expiry, First Out)` removal strategy differs from the :abbr:`FIFO (First In,
+First Out)` and :abbr:`LIFO (Last In, First Out)` strategies, because it targets products for
+removal based on **expiration dates** instead of their warehouse receipt dates.
 
 .. note::
-   To have more information about Expiration date, please have a look at
-   :doc:`the related doc <../../management/lots_serial_numbers/expiration_dates>`.
+   For more information about expiration dates, reference the
+   :doc:`Expiration dates <../../management/lots_serial_numbers/expiration_dates>` document.
 
-By activating *Expiration Dates*, it becomes possible to define different dates on the serial/lot
-numbers to be used in *FEFO*. These dates can be set by going to :menuselection:`Inventory -->
-Master Data --> Lots/Serial Numbers`.
+Once the :guilabel:`Expiration Dates` setting is activated, it's possible to define different
+expiration dates for individual serialized products, as well as lot numbers containing many
+products.
+
+Expiration dates can be entered when validating the received products, or set by going to
+:menuselection:`Inventory app --> Products --> Lots/Serial Numbers`. Click :guilabel:`Create`, enter
+the serial number, and select the product from the drop-down menu. Next, select the expiration date
+in the :guilabel:`Dates` tab. Finally, click :guilabel:`Save`.
 
 .. image:: removal/removal-date.png
    :align: center
-   :alt: view of the removal date for 0000001.
-
-Lots are picked based on their removal date, from earliest to latest. Lots without a removal date
-defined are picked after lots with removal dates.
+   :alt: View of the removal date for 0000001.
 
-.. note::
-   Other dates are for informational and reporting purposes only. If not removed from the stock,
-   lots that are past the expiration dates may still be picked for delivery orders!
-
-To use the *FEFO* strategy, once again go to :menuselection:`Inventory --> Configuration -->
-Product Categories` and choose *FEFO* as the *Force Removal Strategy*.
+Lots are picked based on their removal date, from earliest to latest. Removal dates indicate how
+many days *before* the expiration date, the product needs to be removed from stock. The removal date
+is set on the product form. Lots without a removal date defined are picked after lots with removal
+dates.
 
-.. image:: removal/first-expiry-first-out.png
-   :align: center
-   :alt: view of the FEFO strategy being set up as forced removal strategy.
+.. warning::
+   If products are not removed from stock when they should be, lots that are past the expiration
+   date may still be picked for delivery orders!
 
-For this particular case, let’s use hand cream. As usual, we have three lots of them.
+To test this example using the :abbr:`LIFO (Last In, First Out)` strategy, first navigate to
```

Comment:
Same here---the example should be set up again in this section. Also should be "FEFO" instead of "LIFO" here :)